--- yaz-2.0.21/src/Makefile.am.orig	2004-05-16 00:11:25.138435168 +0200
+++ yaz-2.0.21/src/Makefile.am	2004-05-16 00:13:10.354439904 +0200
@@ -63,7 +63,7 @@
 libyaz_la_LIBADD=$(WRAPLIBS)
 
 libyazthread_la_SOURCES=
-libyazthread_la_LIBADD=thr-nmem.lo thr-statserv.lo thr-eventl.lo
+libyazthread_la_LIBADD=thr-nmem.lo thr-statserv.lo thr-eventl.lo libyaz.la
 libyazthread_la_LDFLAGS=-version-info $(YAZ_VERSION_INFO)
 
 # Rules for Z39.50 V3
