--- yaz-1.9.2/lib/Makefile.am.orig	Tue Oct 22 14:51:18 2002
+++ yaz-1.9.2/lib/Makefile.am	Thu Jan  2 19:32:27 2003
@@ -21,13 +21,13 @@
 # No real sources
 libyaz_la_SOURCES=
 
-libyazthread_la_LIBADD=nmem.lo statserv.lo eventl.lo
+libyazthread_la_LIBADD=nmem.lo statserv.lo eventl.lo -lpthread
 libyazthread_la_SOURCES=
 
 libyazmalloc_la_LIBADD=xmalloc.lo
 libyazmalloc_la_SOURCES=
 
-libyazssl_la_LIBADD=tcpip.lo comstack.lo
+libyazssl_la_LIBADD=tcpip.lo comstack.lo $(SSL_LIBS)
 libyazssl_la_SOURCES=
 
 THREADED_FLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/server @CFLAGSTHREADS@
